docs/html/_geo_bounds_8h_source.html

```diff
@@ -75,7 +75,7 @@
 <div class="ttc" id="aclass_novel_r_t_1_1_geo_bounds_html"><div class="ttname"><a href="class_novel_r_t_1_1_geo_bounds.html">NovelRT::GeoBounds</a></div><div class="ttdef"><b>Definition:</b> GeoBounds.h:9</div></div>
 <!-- start footer part -->
 <hr class="footer"/><address class="footer"><small>
-Generated on Sat Oct 5 2019 18:23:31 for NovelRT by &#160;<a href="http://www.doxygen.org/index.html">
+Generated on Sun Oct 6 2019 17:35:19 for NovelRT by &#160;<a href="http://www.doxygen.org/index.html">
```

Comment:
Is there a way to avoid this timestamp?

It seems non-ideal to have hundreds of one line edits changing the date/time.

Reply:
It was a option I enabled, so yes I can disable it again.